decidim-budgets/lib/decidim/api/budget_type.rb

```diff
@@ -3,7 +3,6 @@
 module Decidim
   module Budgets
     class BudgetType < Decidim::Api::Types::BaseObject
-      implements Decidim::Core::ScopableInterface
```

Comment:
maybe you need to add 
```
implements Decidim::Core::TaxonomizableInterface
```

Reply:
budgets do not have taxonomies directly, projects have. So here I'm only removing the scope